packages/sanes-browser-extension/src/routes/tx-request/index.dom.spec.ts

```diff
@@ -94,13 +95,39 @@ describe("DOM > Feature > Username Registration Request", (): void => {
     },
   ];
 
-  let identityDOM: React.Component;
+  let txRequestDOM: React.Component;
 
   beforeEach(async () => {
-    identityDOM = await travelToTXRequest(requests);
+    txRequestDOM = await travelToTXRequest(requests);
   }, 60000);
 
   it("should show register username request accept view", async (): Promise<void> => {
-    await findRenderedDOMComponentWithId(identityDOM, REQ_REGISTER_USERNAME);
+    await findRenderedDOMComponentWithId(txRequestDOM, REQ_REGISTER_USERNAME);
+  }, 60000);
+});
+
+describe("DOM > Feature > Create Proposal Request", (): void => {
+  const requests: readonly Request[] = [
+    {
+      id: 1,
+      senderUrl: "http://finnex.com",
+      reason: "Test Create Proposal Request",
+      responseData: {
+        tx: getCreateTextResolutionActionTransaction(),
+      },
+      accept: jest.fn(),
+      reject: jest.fn(),
+    },
+  ];
+
+  let txRequestDOM: React.Component;
+
+  beforeEach(async () => {
+    txRequestDOM = await travelToTXRequest(requests);
+  }, 60000);
+
+  fit("should show proper propsal start date", async (): Promise<void> => {
```

Comment:
- `fit` -> `it`
- Do we need async here?
- Please skip return types for unnamed test callbacks